artifactory.py

```diff
@@ -1501,10 +1500,16 @@ class ArtifactoryPath(pathlib.Path, PureArtifactoryPath):
         # see changes in pathlib.Path, slots are no more applied
         # https://github.com/python/cpython/blob/ce121fd8755d4db9511ce4aab39d0577165e118e/Lib/pathlib.py#L952
         _accessor = _artifactory_accessor
+        parser = _ArtifactoryFlavour()
     else:
         # in 3.9 and below Pathlib limits what members can be present in 'Path' class
         __slots__ = ("auth", "verify", "cert", "session", "timeout")
 
+
+    def __init__(self, *args, **kwargs):
+        # supplying keyword arguments to pathlib.PurePath is deprecated
```

Comment:
I assume we should not silently suppress the kwargs

Reply:
Agreed. We should just axe this override, as you're passing via args below anyway.